client/my-sites/site-settings/style.scss

```diff
@@ -586,6 +586,16 @@ button.site-settings__general-settings-set-guessed-timezone.button.is-borderless
 	pointer-events: none;
 }
 
+#site-privacy-settings .inline-support-link {
+	color: var(--color-text-subtle);
+
+	svg {
+		vertical-align: middle;
+		height: 18px;
+		width: 18px;
+	}
```

Comment:
Can we add an hover effect?


https://github.com/Automattic/wp-calypso/assets/779993/cdefbc76-adc7-4e6a-8274-060fb20f13bb